doc/release/release_0.20.rst

```diff
@@ -222,6 +232,10 @@ Bug fixes
   (`#6263 <https://github.com/scikit-image/scikit-image/pull/6263>`_).
 - Fix automatic detection of the color channel in ``skimage.filters.gaussian`` (this behavior is deprecated, see new deprecations)
   (`#6583 <https://github.com/scikit-image/scikit-image/pull/6583>`_).
+- Fix stacklevel of warning in ``skimage.color.lab2rgb``
+  (`#6616 <https://github.com/scikit-image/scikit-image/pull/6616>`_).
+- Fix the order of return values for ``skimage.feature.hessian_matrix`` and raise an error if ``order='xy'`` is requested for images with more than 2 dimensions
+  (`#6624 <https://github.com/scikit-image/scikit-image/pull/6624>`_).
```

Comment:
@scikit-image/core, not sure how big of a problem this is and if we want to act on it but I think this PR actually broke our API without a deprecation cycle. #6624 while raising an error for non-2D-images for  `order='xy'` might make it consistent with `structure_tensor` the following 

```python
import numpy as np
import skimage as ski

img = np.ones((5, 5, 5))
H_elems_xy = ski.feature.hessian_matrix(
    img, sigma=0.1, order="xy", use_gaussian_derivatives=False
)
H_elems_rc = ski.feature.hessian_matrix(
    img, sigma=0.1, order="rc", use_gaussian_derivatives=False
)
for a, b in zip(H_elems_xy, reversed(H_elems_rc)):
    np.testing.assert_array_equal(a, b)
```

passes when I comment out https://github.com/scikit-image/scikit-image/blob/7f0c8fe7c1e47d233f6043bfd569f04efe1b593d/skimage/feature/corner.py#L305-L306

So previously, this worked for users. Now it raises an error message. I think that should have been a deprecation warning?

Reply:
See #6769. I decided to err on the side of caution and quickly put up a PR in case others agree that this needs to be fixed.

Reply:
Actually, see my [comment](https://github.com/scikit-image/scikit-image/pull/6769#issuecomment-1443627988) there — I think that because this only affects >2D images, we should just ignore it. It's accidental that it ever worked.

Reply:
Resolved in https://github.com/scikit-image/scikit-image/pull/6769